extensions/wikia/VideoPageTool/VideoPageAdminSpecialController.class.php

```diff
@@ -300,6 +317,8 @@ protected function getSubpage() {
 	 * @responseParam string msg - result message
 	 */
 	public function getCalendarInfo() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)